README.md

```diff
@@ -81,6 +81,26 @@ standard scenario which can be optionally used with the local development enviro
 > services on the console set `EDC_CATALOG_CACHE_EXECUTION_PERIOD_SECONDS` to a higher value, e.g. 60, for each EDC
 > participant in `system-tests/docker-compose.yml`.
 
+> Note: The container `cli-tools` will turn into the state `healthy` after registering successfully all participants and
+> will keep running as an entrypoint to the services created by Docker compose. This is useful for local development in order
+> to manually check commands against the participants (e.g. `company1`, `company2`, `company3`).
+
+Sample how to enter the container `cli-tools` and test a command manually.
+
+```powershell
+PS C:\wb\edc\agera\mvd> docker exec -it cli-tools bash
+root@22345bf0c595:/app# java -jar registration-service-cli.jar \
```

Comment:
please remove incidental strings (`wb`, `22345bf0c595`)

Reply:
Good approach! Thanks for sharing! Will change all occurrences.